app/controllers/browse_controller.rb

```diff
@@ -36,10 +32,21 @@ def show
 private
 
   def show_html(page)
-    render :show,
-           locals: {
-             page: page,
-           }
+    template = :show
+    if new_browse_variant_b?
+      slimmer_template "gem_layout_full_width"
+      template = :new_show
+    end
+    render template, locals: { page: page }
+  end
+
+  def index_html(page)
+    template = :index
+    if new_browse_variant_b?
+      slimmer_template "gem_layout_full_width"
+      template = :new_index
+    end
+    render template, locals: { page: page }
```

Comment:
These are itching to be refactored to remove the duplication. And the slimmer logic could also be abstracted into a module and included into both controllers. I have not done that because:
- It's easy to read and understand.
- It was a bit buggy to move the slimmer code out, and only include it in the browse controllers. It lead to some weird state persistence between requests so the relevant layout wasn't consistently picked up. Including the module in the application controller avoided this issue, but then the module is available to ALL the controllers unnecessarily.

Reply:
A small amount of duplication is ok and these methods are small and simple. In this case being explicit is good.